codegen/gql_tristate_value/lib/src/value.dart

```diff
@@ -14,6 +14,15 @@ sealed class Value<T extends Object> {
   /// If the value is non-null, it will be serialized as the value.
   const factory Value.present(T? value) = PresentValue<T>;
 
+  /// If the value is non-null, it will be serialized as the value.
+  /// If the value is null. It will not be serialized.
+  factory Value.ofNullable(T? value) {
+    if (value == null) {
+      return AbsentValue();
```

Comment:
can this be const?

Reply:
It should be... My mistake. I am used to having a lint `prefer_const_constructors` 😅  Thanks 🙏

Reply:
yeah I know, gql_pedantic should be modernized at some point ;)